requirements.txt

```
@@ -1,4 +1,4 @@
-urllib3[secure]>=1.19.1,!=1.21
+﻿urllib3[secure]>=1.19.1,!=1.21
```

There is some unwanted character:
```
diff --git a/requirements.txt b/requirements.txt
index 5c30d3a..36cbae8 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-urllib3[secure]>=1.19.1,!=1.21
+<U+FEFF>urllib3[secure]>=1.19.1,!=1.21
```

```suggestion
urllib3[secure]>=1.19.1,!=1.21
```